scripts/bundled_installer.ps1

```diff
@@ -39,8 +39,11 @@ function Write-StepTitle([String] $Message) {
     Write-OutputWithInvertedColors $equals
 }
 Write-StepTitle "I. Installing Python                          "
-powershell "$PSScriptRoot\install-python.ps1"
```

Comment:
This code here says compile and run this script that is inside the quotes.  So if the path has any  special characters, bad things can happen.